admin/listPurchases/listView.html

```diff
@@ -267,6 +274,39 @@
     `;
 }
 
+function markOrderDispatched(id){    
+    let dispatchButton = document.querySelector(".more-info-" + id + " .mark-as-sent-btn");
+    dispatchButton.classList.add("is-clicked");
```

Comment:
Logically, setting a button in a particular state has nothing to do with marking an order as dispatched. I would move these lines of code outside of this function or rename the function to fit what it does. E.g. `onMarkDispatchButtonClick` indicates that it reacts to a button click, and so changing that button's state is more in scope.